Makefile

```diff
@@ -12,8 +12,8 @@ all:: $(TARGETS)
 DECIDE: launch.c headers/decide.h intermediate_results/LICs.c intermediate_results/CMV.c intermediate_results/FUV.c intermediate_results/PUM.c 
 	gcc launch.c headers/decide.h intermediate_results/LICs.c intermediate_results/CMV.c intermediate_results/FUV.c intermediate_results/PUM.c  \
 	-o launch $(CFLAGS)
```

Comment:
I think DECIDE would be a more appropriate name for the final executable, what do you guys think?

Reply:
okay